modules/tinymce/CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Search and replace plugin would incorrectly find matching text inside non-editable root elements. #TINY-10162
 - Search and replace plugin would incorrectly find matching text inside SVG elements. #TINY-10162
 - Removed use of `async` for editor rendering which caused visual blinking when reloading the editor in-place. #TINY-10249
+- Merging an exernal `p` inside a `list` via `delete`/`backspace` would incorreclty try to move a parent element inside a child element. #TINY-10289
```

Comment:
Apperently / is hard to understand for some readers I always get feed back on that since I use that all the time so use or instead. Fixed some typos as well and unwrapped the ticks around delete and backspace since they are more like words then say variables.

```suggestion
- Merging an external `p` inside a `list` via delete or backspace would incorrectly try to move a parent element inside a child element. #TINY-10289
```

Reply:
done